src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -424,11 +450,7 @@ class TransactionDetailsComponent extends React.Component<Props, State> {
 
     // A transaction is acceleratable when it's unconfirmed and has a recorded nonce
     // Disabled until we fix a bug:
-    const isAcceleratable = false
-    // edgeTransaction.spendTargets?.length &&
-    // currencyInfo?.canReplaceByFee === true &&
-    // edgeTransaction.confirmations === 'unconfirmed' &&
-    // edgeTransaction.otherParams?.nonceUsed != null
+    const canAccelerate = currencyInfo?.canReplaceByFee === true && acceleratedTx != null
```

Comment:
Doesn't the new API remove the need for `currencyInfo?.canReplaceByFee`? The `accelerate` method now gives the exact answer.